perf-tests/src/test/scala/sttp/tapir/perf/Simulations.scala

```diff
@@ -3,28 +3,164 @@ package sttp.tapir.perf
 import io.gatling.core.Predef._
 import io.gatling.core.structure.PopulationBuilder
 import io.gatling.http.Predef._
+import sttp.tapir.perf.Common._
 
-import scala.concurrent.duration.{DurationInt, FiniteDuration}
+import scala.concurrent.duration._
+import scala.util.Random
+import io.gatling.core.structure.ChainBuilder
 
 object CommonSimulations {
-  private val userCount = 100
   private val baseUrl = "http://127.0.0.1:8080"
+  private val random = new Random()
 
-  def testScenario(duration: FiniteDuration, routeNumber: Int): PopulationBuilder = {
-    val httpProtocol = http.baseUrl(baseUrl)
-    val execHttpGet = exec(http(s"HTTP GET /path$routeNumber/4").get(s"/path$routeNumber/4"))
+  def randomByteArray(size: Int): Array[Byte] = {
+    val byteArray = new Array[Byte](size)
+    random.nextBytes(byteArray)
+    byteArray
+  }
+
+  def randomAlphanumByteArray(size: Int): Array[Byte] =
+    Random.alphanumeric.take(size).map(_.toByte).toArray
+
+  lazy val constRandomLongBytes = randomByteArray(LargeInputSize)
+  lazy val constRandomLongAlphanumBytes = randomAlphanumByteArray(LargeInputSize)
+
+  def getParamOpt(paramName: String): Option[String] = Option(System.getProperty(s"tapir.perf.${paramName}"))
+
+  def getParam(paramName: String): String =
+    getParamOpt(paramName).getOrElse(
+      throw new IllegalArgumentException(
+        s"Missing tapir.perf.${paramName} system property, ensure you're running perf tests correctly (see perfTests/README.md)"
+      )
+    )
+
+  private lazy val userCount = getParam("user-count").toInt
+  private lazy val duration = getParam("duration-seconds").toInt
+  private val httpProtocol = http.baseUrl(baseUrl)
+
+  // Scenarios
+  val warmUpScenario = scenario("Warm-Up Scenario")
```

Comment:
warmup info would be great to include in the help as well

Reply:
What should be shown there? It is not configurable right now.

Reply:
well you have the "help" screen, it could just mention that there's a 5-second 3-user warmup before the tests are run, configured as below: (list of options)

Reply:
@adamw You mean this screen?
```
[info] ===========================================================================================
[info] Running a suite of 1 tests, each for 1 users and 10 seconds
[info] Servers: List(pekko.Vanilla)
[info] Simulations: List(SimpleGet)
[info] Expected total duration: at least 1 minutes
[info] Generated suite report paths will be printed to stdout after all tests are finished.
[info] ===========================================================================================
```
Let's call it "info screen" to tell it apart from the message displayed when input params are missing/incorrect :)
So the help screen would be:
```
[error] Usage: perf [options]
[error]   -s, --server <value>    Comma-separated list of short server names, or '*' for all. Available servers: http4s.TapirMulti, http4s.Tapir, http4s.VanillaMulti, http4s.Vanilla, netty.cats.TapirMulti, netty.cats.Tapir, netty.future.TapirMulti, netty.future.Tapir, pekko.TapirMulti, pekko.Tapir, pekko.VanillaMulti, pekko.Vanilla, play.TapirMulti, play.Tapir, play.VanillaMulti, play.Vanilla, vertx.TapirMulti, vertx.Tapir, vertx.VanillaMulti, vertx.Vanilla, vertx.cats.TapirMulti, vertx.cats.Tapir
[error]   -m, --sim <value>       Comma-separated list of short simulation names, or '*' for all. Available simulations: PostBytes, PostFile, PostLongBytes, PostLongString, PostString, SimpleGetMultiRoute, SimpleGet
[error]   -u, --users <value>     Number of concurrent users, default is 1
[error]   -d, --duration <value>  Single simulation duration in seconds, default is 10
[error]   -g, --gatling-reports   Generate Gatling reports for individuals sims, may significantly affect total time (disabled by default)
```